### ISSUE TITLE
[Meru - AB] Layout - Top-right sharing button

### ISSUE THREAD

**Issue by philipperolet** [dust-tt/dust#3780]
Details on what to do in the **[Figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4900-40669&mode=design&t=6tNvt0C7wUTsOHc4-0)** (also, see how it works in the [Prototype](https://www.figma.com/proto/QOxHwppG64GtPocpDhS6Y7/Product?page-id=3366%3A31345&type=design&node-id=4900-40669&viewport=1173%2C2546%2C0.27&t=lV2M2VcNHLWO3Bpe-1&scaling=min-zoom&starting-point-node-id=4900%3A40669) + context & discussion in the [making issue](https://github.com/dust-tt/decisions/issues/164))

Note the various [dropdown variants](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4743-20569&mode=design&t=6tNvt0C7wUTsOHc4-4)

Clicking on the button opens the dropdown. Wwithin it, clicking on the chip opens the existing sharing dropdown (see Sharing.tsx)

**Comment by philipperolet** [comment]
The link should point to... @Duncid  what should the link point to? 😅 A conversation starter with the agent (like when clicking new but prefilled with @agentinquestion) ? Or the gallery with the detailed view of said assistant opened?

I also take it this change is for all Sharing dropdowns? i.e. the sharing dropdown from assistant detailed view would be affected too?

The link should point to the stable version of the assistant (not the in-edition one), and as such it should be disabled for assistants not yet created. Correct?

**Comment by Duncid** [comment]
> The link should point to... @Duncid what should the link point to? 😅 A conversation starter with the agent (like when clicking new but prefilled with @agentinquestion) ? Or the gallery with the detailed view of said assistant opened?

In the proposal, I pushed "Assistant Detail View".
So Dust basic app screen + Assistant details drawer opens (show instructions and data sources, access to "try").

Not held strong on my side, new conversation with Assistant pinged on a basic question would work, a "Try" drawer could do too.

**Comment by philipperolet** [comment]
Thanks ! Clear
Sorry for missing it in the proposal, I had a prior misconception that it was part of the sharing dropdown and didn't think to look in nice-to-haves

**Comment by spolu** [comment]
@Duncid is it expected that one can't see the sharing mode of the assistant without clicking on this button when editing the assistant? (tbc I'm fine with it, just want to double check)

**Comment by Duncid** [comment]
> @Duncid is it expected that one can't see the sharing mode of the assistant without clicking on this button when editing the assistant? (tbc I'm fine with it, just want to double check)

Yes that's a consequence of the design indeed.
I think that with Ahuna 2, status of assistants is made very clear before editing — I may be wrong on that.
I'll be thinking about how we can show the status when dropdown is closed.

**Comment by spolu** [comment]
I agree that Ahuna 2 made it much clearer outside of the screen which mitigates the fact that we don't see it here :+1:

**Comment by philipperolet** [comment]
This issue can be done independently at 99%. Only thing with dependency is positioning the button at the end of the tab bar--either by waiting merge of #3776 or by rebasing on it

**Comment by spolu** [comment]
Splitting the addition of the link in a new issue
